src/migrateCalculatedMeasures.ts

```diff
@@ -0,0 +1,68 @@
+import { ContentRecord } from "@activeviam/activeui-sdk";
+import { migrateCalculatedMeasure } from "./migrateCalculatedMeasure";
+
+/**
+ * Returns the converted content record tree for the calculted measures, ready to be used in ActiveUI 5.
+ */
+export function migrateCalculatedMeasures(
```

Comment:
In terms of code split I find it a bit weird that `migrateCalculatedMeasure` just handles migrating the `content` part of the calculated measure, but that the `structure` part is inlined.

I would see it more like that:
```
function migrateCalculatedMeasures(legacyCalculatedMeasures) {
  const contentFolder = {entry: {isDirectory: true, owners: ["ROLE_USER"], readers: ["ROLE_USER"]}, children: {}}
  const structureFolder = {entry: {isDirectory: true, owners: ["ROLE_USER"], readers: ["ROLE_USER"]}, children: {}}
  legacyCalculatedMeasures.forEach(() => {
   // Add what's necessary in contentFolder
   // Add what's necessary in structureFolder
  }
  
  // return the calculated_measures directory owned by CS_ROOT, with content/structure as children
)
}
```